sidecar/Dockerfile

```
@@ -9,8 +11,8 @@ RUN apt-get -qq update \
 && adduser --quiet --system --uid 998 --home /var/lib/postgresql --no-create-home --shell /bin/bash --group postgres \
 && adduser --quiet --system --uid 999 --home /var/lib/xroad --no-create-home --shell /bin/bash --group xroad \
 && adduser xroad sudo \
-&& useradd -m xrd -s /usr/sbin/nologin -p '$6$JeOzaeWnLAQSUVuO$GOJ0wUKSVQnOR4I2JgZxdKr.kMO.YGS21SGaAshaYhayv8kSV9WuIFCZHTGAX8WRRTB/2ojuLnJg4kMoyzpcu1' \
-&& echo "xroad-proxy xroad-common/username string xrd" | debconf-set-selections \
+&& useradd -m ${XROAD_ADMIN_USER} -s /usr/sbin/nologin -p $(python3 -c "import crypt; print(crypt.crypt('${XROAD_ADMIN_PASSWORD}', crypt.mksalt(crypt.METHOD_SHA512)))") \
```

Maybe unnecessarily complex to use python to create a hashed password here, consider something like:
```suggestion
&& useradd -m -s /usr/sbin/nologin ${XROAD_ADMIN_USER} \
&& echo "${XROAD_ADMIN_USER}:${XROAD_ADMIN_PASSWORD}" | chpasswd \
```

---

btw, the `devscripts` dependency (line 8 in Dockerfile) is unnecessary since the packages are installed from a repository.

---

Agree with the changes. Done.